### PR TITLE
Amls 4987 - Supervision/TCSP/HVD content changes

### PR DESCRIPTION
**Pull request by matthewharris1**
Supervision/TCSP/HVD content changes

## How Has This Been Tested?
Unit testing
Manual testing

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/hvd/cash_payment.scala.html [hunk 1]
```diff
@@ -32,9 +32,9 @@
 
     <p>@Messages("hvd.cash.data.payment.summary.details")</p>
     <ul class="list list-bullet">
+         <li>@Messages("hvd.cash.payment.data.summary.details.03")</li>
```
Comment (drewmiley) [inline app/views/hvd/cash_payment.scala.html]: Rename to 01,02,03

File: app/views/hvd/expect_to_receive.scala.html [hunk 2]
```diff
@@ -29,6 +29,13 @@
     title = Messages("hvd.expect.to.receive.title") + " - " + Messages("summary.hvd"),
     heading = header
 ) {
+    <p>@Messages("hvd.cash.data.payment.summary.details")</p>
```
Comment (drewmiley) [inline app/views/hvd/expect_to_receive.scala.html]: Pull out as helper?

File: app/views/hvd/cash_payment.scala.html (hunk 1 above)
Comment (matthewharris1) [reply app/views/hvd/cash_payment.scala.html]: Done

File: app/views/hvd/expect_to_receive.scala.html (hunk 2 above)
Comment (matthewharris1) [reply app/views/hvd/expect_to_receive.scala.html]: Done

**Pull request merged**